include/SDL3/SDL_events.h

```diff
@@ -406,6 +423,21 @@ typedef struct SDL_MouseButtonEvent
     float y;            /**< Y coordinate, relative to window */
 } SDL_MouseButtonEvent;
 
+/**
+ * Mouse raw button event structure (event.rbutton.*)
+ *
+ * \since This struct is available since SDL 3.0.0.
+ */
+typedef struct SDL_MouseRawButtonEvent
+{
+    SDL_EventType type; /**< SDL_EVENT_MOUSE_RAW_BUTTON */
+    Uint32 reserved;
+    Uint64 timestamp;   /**< In nanoseconds, populated using SDL_GetTicksNS() */
+    SDL_MouseID which;  /**< The mouse instance id, SDL_TOUCH_MOUSEID, or SDL_PEN_MOUSEID */
+    Uint8 button;       /**< The mouse button index */
+    Uint8 change;       /**< SDL_PRESSED or SDL_RELEASED */
```

Comment:
```suggestion
    Uint8 state;       /**< SDL_PRESSED or SDL_RELEASED */
```
For consistency with the normal mouse button events